test/test_prerun.py

```diff
@@ -213,6 +213,27 @@ def test_ansible_config_get() -> None:
     assert len(paths) > 0
 
 
+@pytest.mark.parametrize(
+    "default",
+    (
+        (None,),
+        (123,),
+    ),
+)
+def test_ansible_config_get_default(default: object) -> None:
+    """Check that config get returns default when appropriate."""
+    result = prerun.ansible_config_get("NON_EXISTING_OPTION", default=default)
+    assert result is default
+
+
+def test_ansible_config_get_raise() -> None:
+    """Check that config get raise if key is not found."""
+    key = "NON_EXISTING_OPTION"
+    with pytest.raises(KeyError) as exc:
+        prerun.ansible_config_get(key)
+    assert exc.value.args[0] == key
```

Comment:
The correct idiom is to use `match=` arg in `pytest.raises()` and not check it separately.

Reply:
I will keep that in mind, is much cleaner. Anyway, that file was removed so the fix is no longer needed.

Reply:
Okay. AFAIK ansible/ansible avoided `match=` for some time because of a bug in old pytest versions but in general, this is the right way.